### ISSUE TITLE
Paused jobs trigger, are marked pending, block the worker queue

### ISSUE THREAD

**Issue by cjcjameson** [concourse/concourse#267]
Hi there,

When I fly up a new pipeline on my local Mac OS machine within a BOSHlite, I often only want to trigger a specific pathway at first for local development. Therefore, at a "fan out" juncture, where one resource feeds 3 jobs (for example), I'll pause two of them to make sure the other one runs first.

I expect that the immediately downstream job from the not-paused job will trigger next and start working.

What happens after one job finishes is that the two paused jobs are still pending, radiating grey, and the downstream job is also marked pending.

The workaround is to cancel the two pending runs of paused jobs. What happens then is that the expected job runs, and then new pending runs for paused jobs are again enqueued. If the desired pipeline has more steps, it's once again blocked.

I feel like this is a regression from a previous fix. I feel like the update in release v0.53 to default things to paused was a stopgap measure for this issue previously. But that wasn't so great either, and  it introduced other annoyances.

Let me/us know if there's more info you need!

Thanks,
C.J.

**Comment by vito** [comment]
Are you using serial groups?

The change in v0.53 was not a stopgap measure, and sounds unrelated to this.

**Comment by cjcjameson** [comment]
We are using serial groups. All the 4 jobs in my example are in the same serial group: "images".

We are pretty aggressive and strict about serial groups for a few reasons:
- We don't expect these builds to change too often in practice/production. They only cycle frequently when we're re-tooling the pipeline.
- The builds are rather resource intensive. We instituted serialization to try to keep our deployment from falling over (or crashing our BOSHlite, can't quite recall).

I'm blocked trying to get my Vagrantfile fixed right now, but once I've got a local concourse running again, I'll experiment with "what happens when we remove the serial groups?". I hypothesize, as it sounds you do as well, that without serial groups, the paused & pending jobs would not block the worker queue for other jobs' builds.

Mm, very good. Disregard my reference to v0.53
